packages/bot-web-ui/src/stores/transactions-store.js

```diff
@@ -174,35 +174,38 @@ export default class TransactionsStore {
 
     recoverPendingContractsById(contract_id) {
         const { ws } = this.root_store;
+        const token = this.root_store.core.client.accounts[this.root_store.core.client.loginid].token;
```

Comment:
const client = ws.core.client;
const token = client.account[client.loginid].token;

Reply:
@sara-fs thanks, let me check that it is not empty object with token

Reply:
@sara-fs unfortunately 'TypeError: Cannot read properties of undefined (reading 'client')'
so, I left my implementation

Reply:
thanks